subsys/bluetooth/audio/ascs.c

```diff
@@ -133,13 +133,15 @@ void ascs_ep_set_state(struct bt_audio_ep *ep, uint8_t state)
 				ops->disabled(stream);
 			}
 			break;
+		case BT_AUDIO_EP_STATE_RELEASING:
```

Comment:
Indentation seems to be one or two TABs off, from line 134, - may be just Github, but please check.

Reply:
Just Github :)